Azure/communication-ui-library-ios
# [Bugfix] Added a work-around is to make sure the setup view rotates back to portrait if the previous screen was on a different orientation.

## Purpose
Added a work-around is to make sure the setup view rotates back to portrait if the previous screen was on a different orientation.

https://user-images.githubusercontent.com/90668345/161344079-b37d9ff8-cfdd-4413-a3bb-ec8e9ae08f24.mp4


## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code
* Join a call (teams and groups are fine), no remote participant is needed
* in Calling view, rotate to landscape
* in Calling view, turn off wifi/internet connection to simulate an error scenario that would land participant back to setup screen
* In setup view, wait for the rotation to reset back to portrait.
* Follow **What to Check** section 

## What to Check
Verify that the following are valid
* Make sure the setup view has portrait orientation.
* Make sure the setup view camera orientation is locked at portrait, (does not flip orientation when phone orientation is changed)


## Other Information
